### PR TITLE
Migrated remaining tests from os-tests.

### PR DESCRIPTION
**Pull request by narmaku**

### CODE REVIEW

**Comment by jabia99** [comment]
Ci tests are failing for azure. But at first glance, it seems unrelated

File: test_suite/generic/test_generic.py [hunk 1]
```diff
@@ -683,7 +717,8 @@ def __get_kdump_config_by_rhel_version(self, rhel_version):
                 ]
             }
             if rhel_version >= 9.1:
-                conf['sysconfig_kdump'][2] = 'KDUMP_COMMANDLINE_REMOVE="hugepages hugepagesz slub_debug quiet log_buf_len swiotlb cma hugetlb_cma ignition.firstboot"'
+                conf['sysconfig_kdump'][
+                    2] = 'KDUMP_COMMANDLINE_REMOVE="hugepages hugepagesz slub_debug quiet log_buf_len swiotlb cma hugetlb_cma ignition.firstboot"'
```
Comment (jabia99) [inline test_suite/generic/test_generic.py:856]: Is this format intentional? It seems a bit weird

**Review by jabia99 — APPROVED** [review]
LGTM! Just a small comment

File: test_suite/generic/test_generic.py (hunk 1 above)
Comment (narmaku) [reply test_suite/generic/test_generic.py:856]: The linter messed it up. I'll fix it.

**Comment by narmaku** [comment]
We are currently blocked by terraform executor issues and some runners not functioning properly/losing ssh connectivity with GitLab.

**Comment by jabia99** [comment]
Everything passed (cannot test `pub` tests yet). Merging!

**Pull request merged**